bottery/platform/telegram/engine.py

```diff
@@ -127,7 +149,10 @@ def build_message(self, data):
         # TODO: Test if the view returned something or not
         response = await self.get_response(view, message)
 
+        chat_id = message.user.id
+        if not message.chat.type == 'private':
+            chat_id = message.chat.id
```

Comment:
Can you move this `if` to a function and then create a unit test for it? It makes easier to tests and refactor and `message_handler` continues to be a small function with limited responsibilities.

Reply:
> ... and then create a unit test for it?

😍